_posts/2022-11-30-trino-summit-2022-bloomberg-recap.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Trino at Bloomberg"
+author: "Vishal Jadhav, Pablo Arteaga, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/bloomberg.jpg
+---
+
+This post continues [a larger series of posts]({% post_url
+2022-11-21-trino-summit-2022-recap %}) on the Trino Summit 2022 sessions.
+Following the [Trino at Apple talk]({\% post_url
+2022-11-22-trino-summit-2022-apple-recap %}), engineers from Bloomberg shared
+the latest in their additions to Trino. With so many analysts producing huge
+amounts of data, Bloomberg uses Trino to federate it all together. But when
+you have many users with different use cases and resource needs, you need
+something to ensure that the huge workloads don't bully the small ones. Enter
+the Trino Load Balancer, a privacy-aware solution to help maintain high
+availability while still treating data security as the first-class citizen that
+it should be.
+
+<!--more-->
+
+{% youtube ePr-iVQ5ri4 %}
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino-at-Bloomberg.pdf">
+  Check out the slides here!
+</a>
+
+## Recap
+
+Bloomberg collects data, creates experimental data, and ingests data from
+vendors. Then its analysts refine and dig into that data, using whatever their
```

Comment:
Change this block
"Bloomberg collects data, creates experimental data, and ingests data from
vendors. Then its analysts refine and dig into that data, using whatever their.
preferred method is, generating even more diverse data, and then other analysts
want to look at and query that generated data, too. Sound like a data mesh?
That's because it is."

To
"Bloomberg collects data, creates experimental data, and ingests data from vendors. Then its data analysts refine, clean and structure that data, using whatever their preferred method is, thereby generating even more diverse data. Internal teams and clients then want to look at and query that generated data, too. Sound like a data mesh? That's because it is."